### ISSUE TITLE
maybe a bug about package manager 

### ISSUE THREAD

**Issue by jiangwenxuan** [racket/drracket#239]
I install drracket 7.1. My computer os is windows10 and x86-64.
When I click package manager, it becomes.
![qq 20181116183537](https://user-images.githubusercontent.com/32122801/48618300-e68a6180-e9d3-11e8-850b-56c80465f11f.png)

**Comment by inkydragon** [comment]
workaround: change language to English. Everything works well.

**Comment by gfbee** [comment]
Cause: the chinese translations of `(string-constant install-pkg-dependencies-fail)` use a full-width colon character instead of a regular colon, causing a regex not to match in the code at the location indicated below.
```
cadr: contract violation
  expected: (cons/c any/c pair?)
  given: #f
  context...:
   /Applications/Racket v7.1/share/pkgs/gui-pkg-manager-lib/pkg/gui/private/by-source.rkt:534:4:
     adjust-deps method in by-source-panel%
```

**Comment by rfindler** [comment]
I believe this has been fixed. Do you see the error in the nightly builds?

Robby

**Comment by jiangwenxuan** [comment]
> NVM, missed the fix from last November.

thank you

**Comment by inkydragon** [comment]
Test `Snapshot: 20190124-a97b3739a3` no error, package manager works well.

**Comment by bremner** [comment]
Just for the record, this is also fixed in 7.2. Maybe the issue can be closed?

**Comment by jbclements** [comment]
Looks like everyone's happy.
